nixos/roles/nix-remote-builder.nix

```
@@ -27,10 +27,13 @@ in
     ];
 
     # Give restricted SSH access to the build scheduler
-    users.extraUsers.root.openssh.authorizedKeys.keys = map
+    users.extraUsers.builder.openssh.authorizedKeys.keys = map
```

```suggestion
    users.extraUsers.nix-remote-builder.openssh.authorizedKeys.keys = map
```
What do you think to remove ambiguity?